### PR TITLE
Modify test cases which are failing parallel execution

### PR DESCRIPTION
**Pull request by LakshmiGayathri19**
* Addressing the "Known limitation" issue for test cases which failed parallel execution.

### Details:
- Few tests cases failed when parallel execution is enabled.
- These failed test cases are modified in order to execute with out errors.
- After modification, two test cases failed due to inconsistent number of parameters passed to the test case.
- The two test cases which failed due to parameter inconsistency is addressed in this commit.

### Tickets:
 - *ticket-id* - [20919](https://github.com/openvinotoolkit/openvino/issues/20919)

### CODE REVIEW

File: tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_upsampling2d.py [hunk 1]
```diff
@@ -54,7 +54,37 @@ def create_keras_upsampling2d_net(self, input_shapes, input_type, size,
...
+        dict(input_names=["x1"], input_shapes=[[3, 5, 4, 6]], input_type=tf.float32,
+             size=(5, 2), data_format='channels_first', interpolation='nearest'),
+    ]
+
+    @pytest.mark.parametrize("params", test_data_channels_first)
+    @pytest.mark.nightly
+    def test_keras_upsampling2d_channels_first(self, params, ie_device, precision, ir_version,
+                                               temp_dir, use_old_api, use_new_frontend):
+        self._test(*self.create_keras_upsampling2d_net(**params, ir_version=ir_version),
                    ie_device, precision, temp_dir=temp_dir, use_old_api=use_old_api, ir_version=ir_version,
```
Comment (rkazants) [inline tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_upsampling2d.py:89]: please revert these changes. The latest update covers sufficient cases

**Review by rkazants — CHANGES_REQUESTED** [review]
please address one comment

**Comment by rkazants** [comment]
@LakshmiGayathri19, why did you open new PR?:)
It seems to be a duplicate PR. Let us close this one and proceed the work here: https://github.com/openvinotoolkit/openvino/pull/21851

Best regards,
Roman

**Comment by LakshmiGayathri19** [comment]
@rkazants, I was asked to resolve merge conflict and push the changes. So I misunderstood and raised a new PR. I will close this PR and continue working on the [old one](https://github.com/openvinotoolkit/openvino/pull/21851).

**Comment by LakshmiGayathri19** [comment]
Duplicate PR.
